src/components/chat/comment-card.tsx

```diff
@@ -0,0 +1,100 @@
+import { inject, observer } from "mobx-react";
+import React from "react";
+import { BaseComponent } from "../base";
+import DocumentCommentIcon from "../../assets/document-id.svg";
+import SendIcon from "../../assets/send-icon.svg";
+import "./comment-card.scss";
+import "../themes.scss";
+
+interface IProps {
+}
+
+interface IState {
+  commentTextAreaHeight: number | string;
+  commentAdded: boolean;
+  commentText: string;
+}
+
+@inject("stores")
+@observer
+export class CommentCard extends BaseComponent<IProps, IState> {
+
+  constructor(props: IProps) {
+    super(props);
+    this.handleCommentTextAreaChange = this.handleCommentTextAreaChange.bind(this);
+    this.handleCancelPost = this.handleCancelPost.bind(this);
+    this.handleSendPost = this.handleSendPost.bind(this);
+
+    this.state = {
+      commentTextAreaHeight: 35,
+      commentAdded: false,
+      commentText: "",
+    };
+  }
+
+  public render() {
+    const { ui } = this.stores;
+    return (
+      <div className={`comment-card ${ui.activeNavTab}`} data-testid="comment-card">
+        {this.renderThreadHeader()}
+        {this.renderCommentTextbox()}
+      </div>
+    );
+  }
+
+  private renderThreadHeader() {
+    const { ui } = this.stores;
+    return (
+      <div className={`comment-card-header ${ui.activeNavTab}`}>
+        <DocumentCommentIcon className="new-thread-header-icon"/>
+      </div>
+    );
+  }
+  handleCommentTextAreaChange = (event: React.FormEvent<HTMLTextAreaElement>) => {
+    const target = event.currentTarget as HTMLTextAreaElement;
+    const targetText = target.value;
+    if (targetText === "") {
+      this.setState({commentTextAreaHeight: 35, commentAdded: false, commentText: ""});
+    } else {
+      this.setState({commentTextAreaHeight: target.scrollHeight, commentAdded: true, commentText: targetText});
+    }
+  };
+  handleCancelPost = () => {
+    this.setState({commentTextAreaHeight: 35, commentAdded: false, commentText: ""});
+  }
+  handleSendPost = () => {
+    alert(`You are sending this comment: ${this.state.commentText}`);
```

Comment:
```suggestion
    alert(`You are posting this comment: ${this.state.commentText}`);
```
This really doesn't matter since it's temporary UI, but "sending" implies email to me. This is really more of a "post" operation.